ts/pages/governance/governance.tsx

```diff
@@ -65,6 +66,11 @@ export class Governance extends React.Component<RouteComponentProps<any>> {
     }
     public render(): React.ReactNode {
         const { isVoteReceived, tally } = this.state;
+
+        const now = moment();
+        const pstOffset = '-0800';
+        const deadlineToVote = this._proposalData?.voteEndDate?.utcOffset(pstOffset);
+        const hasVoteEnded = deadlineToVote?.isBefore(now) || false;
```

Comment:
none of this data should ever be undefined (but since it does array index lookup to get the vote, technically can be undefined) -- what's the best boolean default here? I leaned toward the vote being expired if we can't find it/issue loading it

Reply:
(we also don't enforce `undefined` strict errors in this TS project, so those can be sneaky)

Reply:
Agreed, better to default to vote not active in case of issues.